### PR TITLE
Basic mobile home page

### PR DESCRIPTION
**Pull request by Deco354**
This page isn't optimised for web/tablet yet.
It contains a basic header and place holder blog posts.

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
+  <title>Senior Swift - Concise weekly blog covering skills you need day to day that employers want</title>
+  <link rel="stylesheet" href="styles.css">
+
+  <link href="https://fonts.googleapis.com/css?family=Montserrat|Playball|Playfair+Display:700" rel="stylesheet">
+</head>
+
+<body class="home-page">
```
Comment (PeterMcK) [inline home.html]: You don't use class home-page in your css. Instead you select a class .body that doesn't exist in your html.
Suggest you use the simple type selector body instead in your css - and remove the home-page class if it's not needed for anything you can't accomplish with the body selector...

File: home.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      </div>
+      <div class="blog-post">
+        <img class="blog-post-image" alt="" src="images/four.svg"/>
+        <a href="home.html" class="blog-post-title">Fourth Blog Post - No this is the longest because it is in fact a
+        stress test to see what happens when we have an overly large title. Do we cut it off, use elipsis or expand.</a>
+        <time datetime="2019-6-08">June 8 2019</time>
+      </div>
+    </div>
+  </section>
+  </div>
```
Comment (PeterMcK) [inline home.html:42]: Too many closing divs?
Also do you need all of these? If a wrapper won't contain anything that's not already inside the element it's wrapping?

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  font-family: "Montserrat", sans-serif;
+  font-size: 16px;
+}
+
+.home-page {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.h1, .h2, .h3 {
```
Comment (PeterMcK) [inline styles.css]: There are no classes of this name in your html: use the type selectors h1 etc (no dot)

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  font-family: "Playfair Display", serif;
+}
+
+.home-page {
```
Comment (PeterMcK) [inline styles.css]: Duplicate rule

File: home.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    <div class="container">
+      <div class="blog-post">
+        <img class="blog-post-image" alt="" src="images/one.svg"/>
+        <a href="home.html" class="blog-post-title">First Blog Post</a>
+        <time datetime="2019-5-31">May 31 2019</time>
+      </div>
+      <div class="blog-post">
+        <img class="blog-post-image" alt="" src="images/two.svg"/>
+        <a href="home.html" class="blog-post-title">Second Blog Post - A longer title</a>
+        <time datetime="2019-6-03">June 3 2019</time>
```
Comment (PeterMcK) [inline home.html]: format: use two digits for month

File: home.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  <link href="https://fonts.googleapis.com/css?family=Montserrat|Playball|Playfair+Display:700" rel="stylesheet">
+</head>
+
+<body class="home-page">
+  <section class="home-banner">
+    <h1 class="blog-title">Swifty</h1>
+    <h2 class="blog-description">Weekly swift articles covering actual useful skills with short concise posts</h2>
+  </section>
+
+  <section class="blog-post-list">
```
Comment (PeterMcK) [inline home.html]: section should have a header

Comment (PeterMcK) [reply home.html]: ..sorry, heading. h2, h3 etc.
Though your page should also have a <header> element! And a <footer> element.

Comment (PeterMcK) [reply home.html]: "...should also have a 'header' element! And a 'footer' element".
I wrote the tags, but text inside lesser-than and greater-than signs seem to be ignored...

Comment (Deco354) [reply home.html]: You can use \` to surround code snippets like this:  `<header>`

Comment (Deco354) [reply home.html]: > section should have a header

Hmm I'd like to just show the posts immediately. I've noticed other sites often have a heading saying "Recent Posts" etc but I feel it speaks for itself.

Do I need to do this for the sake of page scraping/ SEO?

File: home.html (hunk 2 above)
Comment (Deco354) [reply home.html:42]: I've added container as a section class and removed the div.

File: home.html (hunk 6 above)
Comment (Deco354) [reply home.html]: > "...should also have a 'header' element! And a 'footer' element".

I've converted HomeBanner from a `<div>` to a `<header>`. The footer I'm going to do in a future branch as I plan on putting quite a lot of content in it. (social media, git, Stackoverflow links, and a photo). I figure a detailed footer is more effective than an about page no-one will ever click on.

**Pull request merged**